packages/ocaml/ocaml.4.12.0/opam

```diff
@@ -6,9 +6,9 @@ and polls it to initialise specific variables like `ocaml:native-dynlink`"""
 maintainer: "platform@lists.ocaml.org"
 depends: [
   "ocaml-config"
-  "ocaml-base-compiler" {= "4.12.0"} |
-  "ocaml-variants" {>= "4.12.0" & < "4.12.1~"} |
-  "ocaml-system" {>= "4.12.0" & < "4.12.1~"}
+  "ocaml-variants" {>= "4.12.0~" & < "4.12.1~"} |
+  "ocaml-system" {>= "4.12.0" & < "4.12.1~"} |
+  "metaocaml" {>= "4.12.0~" & < "4.12.1~"}
```

Comment:
Well no need, `ocaml-base-compiler` if needed would just be an alias for `ocaml-variants`+`ocaml-options-vanilla` ?
(indeed `metaocaml` is not needed at the moment since the package doesn't exist, on my other branch I moved it to its own package)

Reply:
For now, I think it's better to keep the existing layout - for example, it wouldn't surprise me to find scripts which behave dependent on whether there's ocaml-base-compiler or ocaml-variants, for example?

Reply:
So you mean, duplicating the (actual) compiler definition into an `ocaml-base-compiler` package ?

Reply:
Sort of - but it wouldn't need any of the `configure` switches and presumably should conflict all the `options` packages?